localstripe/resources.py

```diff
@@ -229,6 +229,8 @@ def do_expand(path, obj):
                     do_expand(path, i)
             else:
                 k, path = path.split('.', 1) if '.' in path else (path, None)
+                if obj[k] is None:
+                    return
```

Comment:
It feels a bit strange to me here, instead I propose to set it in the final `if`:

```diff
-                if path is not None:
+                if path is not None and obj[k] is not None:
                     do_expand(path, obj[k])
```

Reply:
Nice, cleaner like that indeed, thanks!